pages/common/dexdump.md

```diff
@@ -0,0 +1,19 @@
+# dexdump
+
+> Display information about Android DEX files.
+
```

Comment:
Is there any document link we can put here?

Reply:
I could not find any official documentation :slightly_frowning_face:

Reply:
Okay.

Reply:
Looks like this conversation never got resolved.  I'll do that now.  Btw @Rudloff it's okay to use non-official links, at least as of 2021. 😉